### PR TITLE
[9.0][ADD] sale_fixed_discount

### PR DESCRIPTION
**Pull request by LoisRForgeFlow**
This module addresses the issues https://github.com/OCA/sale-workflow/issues/521 and https://github.com/OCA/account-invoicing/issues/310

* [x] : Depends on https://github.com/OCA/account-invoicing/pull/311

Sale Fixed Discount
===================

This module extends the functionality of Sales to allow you to apply fixed
amount discount at sales order line level.

The module also extends the invoice lines to show fixed discount and both
sales order and customer invoice reports.

Configuration
=============

To configure this module, you need to:

1. Go to *Sale > Configuration >Settings*.

2. In the *Discount* option select *Allow discounts on sales order lines*.

Usage
=====

To use this module, you need to:

1. Go to *Sales*.

2. Create a Sales Order and specify the type of discount and fixed/percent
   discount in a line.

### CODE REVIEW

**Review by JordiBForgeFlow — APPROVED** [review]

**Comment by JordiBForgeFlow** [comment]
@mreficent @darshan-serpent  can you please review?

File: sale_fixed_discount/README.rst [hunk 1]
```diff
@@ -0,0 +1,69 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (MiquelRForgeFlow) [inline sale_fixed_discount/README.rst]: png

File: sale_fixed_discount/README.rst [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+check there if your issue has already been reported. If you spotted it first,
+help us smash it by providing detailed and welcomed feedback.
+
+Credits
+=======
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
```
Comment (MiquelRForgeFlow) [inline sale_fixed_discount/README.rst]: Icon https://odoo-community.org/logo.png`_.

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

File: sale_fixed_discount/views/sale_order_view.xml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 Eficent Business and IT Consulting Services S.L.
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
```
Comment (MiquelRForgeFlow) [inline sale_fixed_discount/views/sale_order_view.xml:5]: v9 uses still openerp I think

Comment (LoisRForgeFlow) [reply sale_fixed_discount/views/sale_order_view.xml:5]: it uses both and both are valid

**Comment by MiquelRForgeFlow** [comment]
:green_apple:

**Pull request merged**